src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -1,96 +1,82 @@
 @import '~@pxblue/colors/palette.scss';
-
+$duration: 200ms;
+$animationTimingFunction: ease-out;
 section.mat-typography {
     padding: 16px 24px;
     .example-section {
         margin-bottom: 16px;
     }
     .show-text {
         .text-container {
-            opacity: 1;
-            visibility: visible;
-            -webkit-transition: opacity 200ms, visibility 200ms;
-            transition: opacity 200ms, visibility 200ms;
+            display: block;
+            -webkit-animation: fadeInAnimation $duration ease-out;
+            -moz-animation: fadeInAnimation $duration ease-out;
+            -o-animation: fadeInAnimation $duration ease-out;
```

Comment:
the `animation` property does not need to be vendor-prefixed — it is a standard feature on all of the browsers the we support.

Same with the `keyframe` animation definitions befow.

Reply:
Removed prefixes.